WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryViewModel.cs

```diff
@@ -185,24 +186,29 @@ public HistoryViewModel(WalletViewModel walletViewModel, IObservable<Unit> updat
 
 	public void SelectTransaction(uint256 txid)
 	{
-		var txnItem = Transactions.FirstOrDefault(item =>
-		{
-			if (item is CoinJoinsHistoryItemViewModel cjGroup)
-			{
-				return cjGroup.CoinJoinTransactions.Any(x => x.TransactionId == txid);
-			}
+		// We unselect any previous selection. Otherwise, if the item is already selected, it won't flash again.
+		Source.RowSelection!.SelectedIndex = IndexPath.Unselected;
```

Comment:
Since this method can be called any value of `uint256` I think the row should only be cleared once the transaction that should be highlighted is found.